native/cocos/engine/Engine.cpp

```diff
@@ -331,15 +331,19 @@ void Engine::onPause() {
     AppEvent appEv;
     appEv.type = AppEvent::Type::PAUSE;
     dispatchEventToApp(OSEventType::APP_OSEVENT, appEv);
-
+#if CC_PLATFORM == CC_PLATFORM_WINDOWS
+    cc::EventDispatcher::dispatchDestroyWindowEvent();
+#endif
```

Comment:
Is pause the same as destroy window?

Reply:
> Is pause event the same as destroy window event?


Unlike the destroy event, it is only used in the render module. This was done in 3.4, I just reused the call